app/views/cases/search_filters/_final_deadline.html.slim

```diff
@@ -0,0 +1,51 @@
+= form_for @query, url: search_cases_path, method: :get do |f|
+  = hidden_field_tag('search_query[filter_type]', 'final_deadline')
+  = hidden_field_tag('search_query[parent_id]', @parent_id)
+  fieldset
+    legend
+      .bold-medium
+        = "Filter by final deadline dates"
+
+    .grid-row
+      .column-full
+        .js-quick-links
+          span#quick-link-label.bold-small
+             = "Due"
+          ul.quick-link-options
+            li.quick-link-option--first
+              - today = {day: @query.available_deadlines[:today].strftime("%d"), month: @query.available_deadlines[:today].strftime("%m"), year: @query.available_deadlines[:today].strftime("%Y")}.to_json
```

Comment:
Can't we set these values in the action or service? I don't think the view is the best place to have them.